### DISCUSSION TITLE
Can organization ownership be retained if I recreate a user with the same unlinked email address?

### DISCUSSION THREAD

**Discussion by matsuo-haruhito** [community/community#146715] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am unable to log in to my account due to issues with 2FA.
To resolve this, I’m considering unlinking the email address from the account and then recreating the user with the same email address.

However, I couldn’t find clear information in the documentation about whether this would affect organization ownership, especially if the account in question is the sole owner of an organization.

Would recreating the account with the same email address transfer organization affiliation and ownership properly? Or would this cause any issues?

**Comment by Ravindulakmina** [comment, marked as answer]
You are better off seeking support on Github.

**Comment by Samele1234** [comment]
May I ask if this issue has been resolved?

**Reply by matsuo-haruhito** [reply]
Sorry, I hadn’t checked at all.

I couldn’t make any changes to the organization structure.
I managed to do it by creating a new organization and changing the owner of each repository to the new organization.
There were about 250 repositories, so it was a pain. lol
